Sphinx docs are failing due to non-ASCII character in references.bib

A stray non-ASCII character in `references.bib` added in https://github.com/pandegroup/openmm/pull/1942 is [causing the `dev` conda package builds to fail](https://travis-ci.org/omnia-md/conda-dev-recipes/jobs/309839769#L8989-L9013):
```
! Package inputenc Error: Unicode char ϕ (U+3D5)
(inputenc)                not set up for use with LaTeX.
See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.9879 ...ing improved sampling of the backbone ϕ
                                                  , \(\psi\) and side-chain ...
? 
! Emergency stop.
 ...                                              
                                                  
l.9879 ...ing improved sampling of the backbone ϕ
                                                  , \(\psi\) and side-chain ...
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on OpenMMUsersGuide.log.
make[5]: *** [OpenMMUsersGuide.pdf] Error 1
make[4]: *** [latexpdf] Error 2
make[3]: *** [sphinx-docs/userguide/latex/OpenMMUsersGuide.pdf] Error 2
make[2]: *** [docs-source/CMakeFiles/sphinxpdf.dir/all] Error 2
make[1]: *** [docs-source/CMakeFiles/sphinxpdf.dir/rule] Error 2
make: *** [sphinxpdf] Error 2
```
Here's the failure in `docs-source/usersguide/references.bib`:
```
@article{Best2012,
   author = {Best, Robert B. and Zhu, Xiao and Shim, Jihyun and Lopes, Pedro E. M. and Mittal, Jeetain and Feig, Michael and MacKerell, Alexander D.},
   title = {Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone ϕ, ψ and Side-Chain χ1 and χ2 Dihedral Angles},
```
I'll open a PR to correct this, but @peastman should add a travis test branch that will fail if the sphinx docs builds fail so we don't merge PRs that break the conda builds.

--- a/.travis.yml
+++ b/.travis.yml
@@ -183,7 +183,7 @@ script:
   - if [[ ! -z "${DOCS_DEPLOY}" && "${DOCS_DEPLOY}" = "true" ]]; then
       pip install sphinx==1.5.6 sphinxcontrib-bibtex sphinxcontrib-lunrsearch sphinxcontrib-autodoc_doxygen;
       make sphinxhtml;
-      make sphinxpdf; # build PDF for testing
+      make sphinxpdf;
       make C++ApiDocs PythonApiDocs;
       mkdir -p api-docs;
       mv sphinx-docs/userguide/html api-docs/userguide;

--- a/.travis.yml
+++ b/.travis.yml
@@ -183,6 +183,7 @@ script:
   - if [[ ! -z "${DOCS_DEPLOY}" && "${DOCS_DEPLOY}" = "true" ]]; then
       pip install sphinx==1.5.6 sphinxcontrib-bibtex sphinxcontrib-lunrsearch sphinxcontrib-autodoc_doxygen;
       make sphinxhtml;
+      make sphinxpdf; # build PDF for testing
       make C++ApiDocs PythonApiDocs;
       mkdir -p api-docs;
       mv sphinx-docs/userguide/html api-docs/userguide;

--- a/docs-source/usersguide/references.bib
+++ b/docs-source/usersguide/references.bib
@@ -41,7 +41,7 @@
 
 @article{Best2012,
    author = {Best, Robert B. and Zhu, Xiao and Shim, Jihyun and Lopes, Pedro E. M. and Mittal, Jeetain and Feig, Michael and MacKerell, Alexander D.},
-   title = {Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone ϕ, ψ and Side-Chain χ1 and χ2 Dihedral Angles},
+   title = {Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone $\phi$, $\psi$, and Side-Chain $\chi_1$ and $\chi_2$ Dihedral Angles},
    journal = {Journal of Chemical Theory and Computation},
    volume = {8},
    number = {9},

